packages/graphql-transformers-e2e-tests/src/__tests__/PredictionsTransformerTests.e2e.test.ts

```diff
@@ -115,7 +115,7 @@ test('test translate and convert text to speech', async () => {
 });
 
 test('test translate text individually', async () => {
-  const germanTranslation = 'Dies ist ein Sprachtest';
+  const germanTranslation = /ist ein Sprachtest$/;
```

Comment:
it's probably better to use regex like /((\bDies\b)|(\bdas\b)|(\bder\b)) ist ein Sprachtest/i;